dashboard/lib/tasks/seed.rake

```diff
@@ -286,7 +287,8 @@ namespace :seed do
     update_scripts(incremental: true)
   end
 
-  timed_task_with_logging scripts_ui_tests: SCRIPTS_DEPENDENCIES do
+  # moved course offerings ui test seed after regular course offerings are seeded
```

Comment:
please expand on this to explain why this is important (basically what you said in the PR description)